README.md

```diff
@@ -105,4 +105,31 @@ You will also need to configure the following environment variables to deploy to
 ```bash
 GIT_USER_NAME: The deploy account's name
 GIT_USER_EMAIL: The deploy user's email
-```
\ No newline at end of file
+```
+
+## Supabase
+
+### Password Reset When Forgotten
+If users forgot their password, they can request for a reset. Atomic CRM handle it for you. All the magic is done by the custom route `/forgot-password` and `/reset-password` that you can find in the `CRM` component.
+
+If you want to update default configuration: 
+
+1. Go to your `config.toml` file
+2. In `[auth]` section set `site_url` to  your application URL
+3. In `[auth]`, add the following URL in the `additional_redirect_urls = [{APPLICATION_URL}}/auth-callback"]`
```

Comment:
This will work for local environment, but how did you manage the prod environment?